.changeset/techdocs-eight-camels-hear.md

```diff
@@ -0,0 +1,8 @@
+---
+'@backstage/plugin-techdocs': minor
+---
+
+Add feedback link icon in Techdocs Reader that directs to Gitlab or Github repo issue page with prefilled title and source link.
+For link to appear, requires repo_url and edit_uri to be filled in mkdocs.yml, as per https://www.mkdocs.org/user-guide/configuration.
```

Comment:
edit_uri does not seem to be required for GitHub, Bitbucket, and GitLab (at least for master branch for GitHub and GitLab, and default branch for Bitbucket) as per the note in https://www.mkdocs.org/user-guide/configuration/#edit_uri

Reply:
edit_uri apparently needs repo_url to have the common hostnames in order to be set automatically (eg gitlab.com, github.com). Self-hosting with other hostnames need a edit_uri specified.

Reply:
Added note on above.